testdata/config/listcheck.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
   name: checkwl
   namespace: istio-config-default
 spec:
-  match: destination.labels["app"] == "ratings"
+  match: (destination.labels["app"]|"unknown") == "ratings"
```

Comment:
Please add a comment above the match condition that says why we are using this expresssion. This is the only example of this type of expression in the "match"

Reply:
happy to add any comment, just tell me the text you'd like to see

Reply:
If an attribute could be potentially absent, use the '|' operator to provide a default value. In the above example if 'destination.labels' is absent, "unknown" is used in its place.

Reply:
cool - I added  (this happens in non kubernetes env) and changed above/below